### PR TITLE
fix(theme): include theme tokens back in vars

### LINKED ISSUE #1232: Variables in 'theme' need to move back to 'vars'
We released a breaking change when working on the experimental theme by moving the color variables out of `vars` and into `theme` instead.

Need to revert this and move them back into the `vars` file.

screen shot 2018-10-10 at 8 38 43 pm

Related PR that wasn't merged https://github.com/IBM/carbon-components/pull/1132
Related Issue https://github.com/IBM/carbon-components/issues/1129

cc @AsthaJain1

### PR DESCRIPTION
**Pull request by alisonjoseph**
Closes #1232

fix to make sure variables are imported back in to vars file

### CODE REVIEW

File: src/globals/scss/_theme-tokens.scss [hunk 1]
```diff
@@ -0,0 +1,222 @@
+@import 'feature-flags';
+@import 'import-once';
+@import 'functions';
+
```
Comment (alisonjoseph) [inline src/globals/scss/_theme-tokens.scss:6]: We need to add the following imports to this file @asudoh 

```
@import 'colors';
@import 'spacing';
```

Comment (asudoh) [reply src/globals/scss/_theme-tokens.scss:6]: @alisonjoseph Good catch - Thanks! Added them.

**Comment by alisonjoseph** [comment]
LGTM! I can't approve though since I open the PR  ✅ 

Also isn't letting me merge the latest changes from master in for some reason.

**Review by asudoh — APPROVED** [review]
Signing off for @alisonjoseph - Thank you for reviewing @alisonjoseph!

**Review by joshblack — APPROVED** [review]

**Pull request merged**